### ISSUE TITLE
Can't set marker opacity if also setting marker symbol

### ISSUE THREAD

**Issue by slishak** [plotly/dash-core-components#203]
In the following example, the marker opacity is always 1 if I set a marker symbol. For the second trace, I haven't set a marker so the marker opacity is respected. The legend is drawn with the correct opacity. If I use plotly.offline.plot to plot the figure outside Dash it looks ok.

```python
import dash
import dash_core_components as dcc
import dash_html_components as html
import random

app = dash.Dash()

n_points = 50

x = list(range(n_points))
data = [
    {
        'x': x,
        'y': [random.random() for _ in range(n_points)],
        'marker': {'symbol': 'x', 'opacity': 0.05, 'size': 20},
        'mode': 'markers',
        'type': 'scattergl',
    },
    {
        'x': x,
        'y': [random.random() for _ in range(n_points)],
        'marker': {'opacity': 0.05, 'size': 20},
        'mode': 'markers',
        'type': 'scattergl',
    },
]
fig = {'data': data, 'layout': {}}

app.layout = html.Div(children=[
    dcc.Graph(
        id='example-graph',
        figure=fig,
    )
])


if __name__ == '__main__':
    app.run_server(debug=True)

```

![image](https://user-images.githubusercontent.com/12515343/40229341-3ec17ac0-5a94-11e8-9687-8a7b059f505e.png)

**Comment by bcdunbar** [comment]
@slishak What version are you using? 

I ran the above and marker opacity and symbol work as expected.
```
dash 0.21.1
dash-core-components 0.23.0
dash-html-components 0.11.0
plotly 2.7.0
```

**Comment by slishak** [comment]
Sorry, should have written the versions in the issue. This appears to have been fixed by `dash-core-components 0.23.0`. The issue above occurres (in Chrome) with:
```dash==0.21.1
dash-core-components==0.22.2
dash-html-components==0.11.0
dash-renderer==0.13.0
plotly==2.7.0
```
